src/components/Integrations/Form/IntegrationTypeHttpForm.tsx

```diff
@@ -23,18 +23,16 @@ export const IntegrationTypeHttpForm: React.FunctionComponent<IntegrationTypeFor
                 name="sslVerificationEnabled"
                 ouiaId={ ouiaIdConcat(props.ouiaId, 'is-ssl-verification-enabled') }
             />
-            { props.type !== IntegrationType.ANSIBLE &&
```

Comment:
Could you remove the import of `IntegrationType` ? 

The linter is complaining - thanks!

Reply:
done